resources/local/config.toml

```diff
@@ -277,6 +277,13 @@ qps_limit = 100
 # Maximum number of bytes to accept in a single request body.
 max_body_bytes = 2_621_440
 
+# Specifies which origin will be reported as allowed by RPC and REST server.
```

Comment:
This comment should be specific to each of the 4 servers.

Reply:
Fixed in 27703947ed34b26e1cab818915f02346210e9499